alacritty.yml

```diff
@@ -211,39 +211,39 @@
 
   # Normal colors
   #normal:
-  #  black:   '#000000'
-  #  red:     '#d54e53'
-  #  green:   '#b9ca4a'
-  #  yellow:  '#e6c547'
-  #  blue:    '#7aa6da'
-  #  magenta: '#c397d8'
-  #  cyan:    '#70c0ba'
-  #  white:   '#eaeaea'
+  #  black:   '#1d1f21'
+  #  red:     '#cc6666'
+  #  green:   '#b5bd68'
+  #  yellow:  '#f0c674'
+  #  blue:    '#81a2be'
+  #  magenta: '#b294bb'
+  #  cyan:    '#8abeb7'
+  #  white:   '#c5c8c6'
 
   # Bright colors
   #bright:
   #  black:   '#666666'
```

Comment:
What is this based on? I'd assume it's not using the Tomorrow Night Bright black since that's darker than the normal's black, but I'd be curious why you've kept it at #666666. Just because you didn't know what better value to change it on, or is this actually a good choice based on some criteria?

Reply:
I wasn't sure what is color is better here and I think it looks decent.

Reply:
It's less than all the foreground colors and more than all the other blacks, so should be good!